src/internet_identity_interface/src/lib.rs

```diff
@@ -227,6 +228,37 @@ pub struct ArchiveConfig {
     pub entries_fetch_limit: u16,
 }
 
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct ActiveAnchorStatistics {
+    // Stats for the last completed collection period for daily and monthly active anchors
+    pub completed: CompletedActiveAnchorStats,
+    // ongoing periods for daily and monthly active anchors
+    pub ongoing: OngoingActiveAnchorStats,
+}
+
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct CompletedActiveAnchorStats {
+    pub daily_active_anchors: Option<ActiveAnchorCounter>,
+    pub monthly_active_anchors: Option<ActiveAnchorCounter>,
+}
+
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct OngoingActiveAnchorStats {
+    // Ongoing active anchor counter for
+    pub daily_active_anchors: ActiveAnchorCounter,
+    // Monthly active users are collected using 30 day sliding windows.
+    // This vec contains up to 30 30-day active windows each offset by one day.
```

Comment:
the comment in .did-file talks about up to 29 entries, not 30 -- is this intentional?